pennylane/capture/autograph/transformer.py

```diff
@@ -119,15 +119,66 @@ def get_cached_function(self, fn):
 def run_autograph(fn):
     """Decorator that converts the given function into graph form.
 
+    AutoGraph can be used in PennyLane's capture workflow to convert Pythonic control flow to PennyLane
+    native control flow. This requires the ``diastatic-malt`` package, a standalone fork of the AutoGraph
+    module in TensorFlow (`official documentation <https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/autograph/g3doc/reference/index.md>`_
+    ).
```

Comment:
Would it be best to link to the same file in the diastatic malt repo? Since the TF one might be out of sync with the DM one

Reply:
Ooh, yes, good point! I'll update that.